jobs/tests.py

```diff
@@ -1,3 +1,28 @@
 from django.test import TestCase
 
-# Create your tests here.
+from model_mommy import mommy
+
+from jobs.models import Job
+
+
+class JobMethodTests(TestCase):
+
+	def setUp(self):
+		self.job = mommy.make(Job, ready_to_publish=True)
+
+	def test_is_published(self):
+		published_date_is_not_none = False
+		self.job.publish()
+		if self.job.published_date:
+			published_date_is_not_none = True
+		self.assertEqual(published_date_is_not_none, True)
+		print "Published date:{0}".format(self.job.published_date)
+
+	def test_expiration_date(self):
+		expiration_date_is_not_none = False
+		self.job.publish()
+		self.job.set_expiration_date()
+		if self.job.expiration_date:
+			expiration_date_is_not_none = True
+		self.assertEqual(expiration_date_is_not_none, True)
+		print "Expiration date:{0}".format(self.job.expiration_date)
```

Comment:
Could be simplified similarly:
```
self.job.publish()
self.assertFalse(self.job.expiration_date)
self.job.set_expiration_date()
self.assertTrue(self.job.expiration_date, "Job has no expiration date")
```
You could also check that the expiration date is what you expect (find self.job.expiration_date against timezone.now().  This gives you a timedelta object that will be in days, mins and secs - do .days on the timedelta object to get the number of days - it should be 59